lib/node_modules/@stdlib/blas/base/cscal/src/cscal_f.c

```diff
@@ -26,34 +26,34 @@
 * Scales a single-precision complex floating-point vector by a single-precision complex floating-point constant.
 *
 * @param N        number of indexed elements
-* @param ca       scalar constant
-* @param CX       input array
-* @param strideX  CX stride length
+* @param alpha       scalar constant
+* @param X       input array
```

Comment:
```suggestion
* @param alpha    scalar constant
* @param X        input array
```

@ShabiShett07 You need to remember to realign descriptions.